content/en/templates/data-templates.md

```diff
@@ -114,6 +114,21 @@ You can use the following code to render the `Short Description` in your layout:
 
 Note the use of the [`markdownify` template function][markdownify]. This will send the description through the Markdown rendering engine.
 
+## Keys as variables
+
+You might want to access a key name as a variable.
+To do so, use the [`index` function][index] with the variable as the second parameter.
+```plaintext
```

Comment:
Markdown style suggestion to match the rest of the file... (Seems like they're not using language names on their fenced code blocks, either, which seems odd. I like that you provided one, although I bet calling these two `go` would get nice highlighting. :) )

````suggestion
You might want to access a key name as a variable. To do so, use the [`index` function][index] with the variable as the second parameter.

```plaintext
````

Reply:
Also: need to define the link target for `[index]` at the bottom of the file with the others.